src/components/Invoice.tsx

```diff
@@ -0,0 +1,115 @@
+import { Grid, TextField, Button, Typography, Box, IconButton } from '@mui/material';
+import { LocalizationProvider, DatePicker } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+
+import SaveIcon from '@mui/icons-material/Save';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+export function Invoice() {
```

Comment:
Mixed exports, you are using named export here, and default export in `Invoice-list.tsx`